docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
   web:
     build:
       context: .
-      dockerfile: web.Dockerfile
+      dockerfile: docker/web/Dockerfile
```

Comment:
Opted for a docker file structure similar to this response: https://stackoverflow.com/a/48243640